doc/api/https.md

```diff
@@ -52,7 +52,7 @@ See [`http.Server#keepAliveTimeout`][].
 <!-- YAML
 added: v0.3.4
 -->
-- `options` {Object} Accepts `options` from [`tls.createServer()`][] and [`tls.createSecureContext()`][].
+- `options` {Object} Accepts `options` from [`tls.createServer()`][], [`tls.createSecureContext()`][] and [`http.createServer()`][].
```

Comment:
nit: long line here. can you wrap at <= 80 chars

Reply:
Good catch, thanks!